src/app/home/options/options.component.ts

```diff
@@ -311,6 +313,7 @@ getJob(job_id: string, API_URL: string) {
       takeWhile(response => !(jobComplete = (response.status === 'completed')), true),
       // Once the stream completes, do any cleanup if necessary
       finalize(() => {
+        this.notificationService.playDing();
```

Comment:
Couldn't test that it plays it after a successful generation because it always throws an error. But I checked that the sound itself worked and this seemed like the most obvious location to put it.

Reply:
We also might want to put this or a different sound into the error handler code at the end of this file to notify user of an error.